build.gradle

```diff
@@ -195,20 +202,37 @@ project(':kcbq-connector') {
         compile (
                 project(':kcbq-api'),
 
-                "com.google.cloud:google-cloud:$googleCloudVersion",
+                "com.google.cloud:google-cloud-bigquery:$googleCloudVersion",
+                "com.google.cloud:google-cloud-storage:$googleCloudVersion",
                 "com.google.auth:google-auth-library-oauth2-http:$googleAuthVersion",
-                "com.google.code.gson:gson:$googleCloudVersion",
+                "com.google.code.gson:gson:$googleCloudGsonVersion",
                 "io.debezium:debezium-core:$debeziumVersion",
+
+                "io.confluent:kafka-connect-avro-converter:$ioConfluentVersion",
+                "io.confluent:kafka-schema-registry-client:$ioConfluentVersion",
+                "org.apache.avro:avro:$avroVersion",
                 "org.apache.kafka:connect-api:$kafkaVersion",
                 "org.apache.kafka:kafka-clients:$kafkaVersion",
                 "org.slf4j:slf4j-api:$slf4jVersion",
                 "org.slf4j:slf4j-simple:$slf4jVersion",
+                "com.jayway.jsonpath:json-path:$jsonPathVersion",
+                "ch.qos.logback:logback-core:$logbackVersion",
+                "log4j:log4j:$log4jVersion"
         )
 
         testCompile (
-                "junit:junit:$junitVersion",
+                "kafka-connect-bigquery:kcbq-connector:main",
+
+                "ch.qos.logback:logback-classic:$logbackVersion",
+                "org.junit.jupiter:junit-jupiter-api:$junitjupiterVersion-M2",
+                "org.junit.platform:junit-platform-runner:$junitPlatformVersion-M4",
                 "org.mockito:mockito-core:$mockitoVersion"
         )
+
+        testRuntime (
+                "org.junit.vintage:junit-vintage-engine:$junitVintageVersion-M4",
+                "org.junit.jupiter:junit-jupiter-engine:$junitjupiterVersion-M4",
```

Comment:
it doesn't look like we need to add these deps, from what I see...

Reply:
Sorry it's confusing... This pr is branched off based on the filter/sanitize pr and include the changes in that pr...